README.md

```diff
@@ -1 +1,100 @@
-# my-portforlio-project
\ No newline at end of file
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 [Hello microverse]
+Describe your project in 1 or 2 sentences.
+
+[Portforlio project] is a project that is used to practice mobile first design in software development
+
+🛠 Built With
+* HTML, CSS
+
+Client
+Server
+Database
+Key Features
+
+
+[Linters]: This is a cool tool that helps avoid coding error
+[package.json]: Holds all the dependencies of our project
+[stylelintrc]: Helps avoid errors in css file
+[HTML]: is a template used to structure our website
+
+🚀 Live Demo
+
+
+💻 Getting Started
+
+This are the tools ylu need to get started
+* A working internet
+* linters
+* A vs code editor
+
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+* A Vs code
+* Understanding Git commands
+* have some knowledge of html and CSS 
+* Basic understanding of linters
+
+
+Setup
+Clone this repository to your desired folder using: 
+git clone https://github.com/Developer-Kingz/Hello-Microverse.git
+move to your desired folder: cd my-folder
+setup linters
+create your html and css files
+
+Install
+Install this project with:
+* install this project with npm install
+* add .hintr and stylelintrc to your package.json
+
+Usage
+To run the project, execute the following command:
+npm install to get the node_modules
+
+👥 Authors
+Kingsley Chidozie Igbor
+
+* Github: @Developer-Kingz
+* LinkedIn: https://www.linkedin.com/in/kingsley-igbor-14b04720b/
+* Twitter: @DaKingzMan
+
+🔭 Future Features
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+🙏 Acknowledgments
+I want to thank the microverse team, for the awesome lesson and also to my code reviewer i say thank you
+
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
+
```

Comment:
Kindly make sure you have the MIT license in your README as is the case in the template provided by microverse.
YOu may check [here](https://github.com/microverseinc/readme-template#:~:text=to%20top)-,License,-This%20project%20is) to learn more how do to include the LICENSE in your repo

Reply:
As requested previously ☝🏽 make sure to add a valid LICENSE link here so that people can know how to use your repository. 📁